lib/persisted-model.js

```diff
@@ -654,6 +676,22 @@ module.exports = function(registry) {
 
     setRemoting(PersistedModel, 'replaceOrCreate', replaceOrCreateOptions);
 
+    setRemoting(PersistedModel, 'upsertWithWhere', {
+      aliases: ['patchOrCreateWithWhere'],
+      description: 'Update an existing model instance or ' +
+      'insert a new one into the data source based on the ' +
+      'where criteria.',
```

Comment:
Yes weird, should be automated. Upon a quick look in our configs, it is set properly. @mountain1234585 Did you run the linter/tests before recommiting the PR?